src/node/delay.rs

```diff
@@ -418,6 +418,11 @@ impl AudioProcessor for DelayReader {
 
         let mut playback_infos = [PlaybackInfo::default(); RENDER_QUANTUM_SIZE];
 
+        // A DelayNode in a cycle is actively processing only when the absolute
+        // value of any output sample for the current render quantum is greater
+        // than or equal to 2−126.
+        let mut is_actively_processing = false;
```

Comment:
Such a strange requirement of the spec. When will this ever be the case? And why not do this for all nodes that process inputs? Ah well..

Reply:
Good points

Reply:
Coming back to this, the smallest possible f32 value is 1.17549435E-38f32 which is exactly 2^-126 (which is not the same as 2E-126!)
https://doc.rust-lang.org/std/primitive.f32.html#associatedconstant.MIN_POSITIVE
So the spec just tells us it is fine to clip to zero when the value is too small to be represented by a f32. So we can drop the check

Reply:
I went with c3439f99619f5d2229ca041982b5

Reply:
nice catch!